src/index.css

```diff
@@ -1,7 +1,25 @@
-@import url("https://chunksui.netlify.app/css/style.css");
+/* @import url("https://chunksui.netlify.app/css/style.css"); */
+@import url("https://unpkg.com/mockman-js@latest/dist/style.css");
+
+/* Overriding styles from component library */
```

Comment:
I thin it'd be better if you imported the stylesheets in the HTML itself

Reply:
For now I'm keeping it here